.github/workflows/test-beta.yml

```diff
@@ -24,4 +24,7 @@ jobs:
       - run: rustup component add rustfmt clippy
       - run: cargo fetch
       - run: cargo clippy -p rioterm-beta -- -D warnings
-      - run: cargo test -p rioterm-beta
+      - name: Install cargo-nextest
+        uses: taiki-e/install-action@nextest
```

Comment:
Adding nextest as a dev dependency would allow it to be cached by the `Swatinem/rust-cache` action, increasing the speed even further 🚀  (I think) .